nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# View all tags

# Description
Added an Admin Tools tab in the navbar housing tag and category management tabs. Displaying all tags in alphabetical order when navigating to /tags from tag management.
Fixes #13 
## Type of change
# Testing Instructions
- Start web API in VS
- Run npm start in client folder to start app
- Login with authorized user
- Navigate to tag management tab in admin tools, will be routed to /tags
- Once in /tags, all tags should be displayed in alphabetical order
- Try hitting the /tags route once logged out to verify route protection
# Checklist:
